### PR TITLE
AOM-43: Implement UI feedback from sprint demo

### PR DESCRIPTION
**Pull request by kingisaac95**
## JIRA TICKET NAME:
AOM-43: [Implement UI feedback from sprint demo]()
### SUMMARY:
* Change delete toastr to an alert box
* Mention app name in alert for successful upload
* Reduce setting button size and reduce space after breadcrumb also reduce space between fields and search box
* Placeholder for search text filed should be: “Search for add-ons to install or clear field to display installed add-ons”
* Increase font size for Header
* Make delete modal vertically centered

### CODE REVIEW

**Comment by kingisaac95** [comment]
@andela-jomadoye @wanjikum @paulupendo You guys should please review this PR

**Comment by kingisaac95** [comment]
@andela-jomadoye check now. Download the file [here](https://issues.openmrs.org/browse/AOM-43) and test it on modules-refapp

File: app/css/openmrs-addonmanager.css [hunk 1]
```diff
@@ -83,6 +83,21 @@ header>ul, a {
     color: #eeeeee;
     padding-right: 2px;
     margin-top: 15px;
+    font-size: 16px;
+}
+
+#curUser {
```
Comment (paulupendo) [inline app/css/openmrs-addonmanager.css]: We could rename this to better define what it does or what it is for and use the naming convention we currently follow. Something like 'account-styling'

**Comment by jomadoye** [comment]
@kingisaac95 I can't find the file to download?

**Review by paulupendo — COMMENTED** [review]
Good Job

File: app/css/openmrs-addonmanager.css (hunk 1 above)
Comment (kingisaac95) [reply app/css/openmrs-addonmanager.css]: Yeah, makes sense

**Comment by kingisaac95** [comment]
@dkayiwa can you look into this please?

**Pull request merged**